src/input.rs

```diff
@@ -102,8 +109,9 @@ pub trait Input: Sized {
     ///
     /// ("touch", "foo").run();
     /// ```
+    #[track_caller]
```

Comment:
It's probably a bit unconventional to have `#[track_caller]` in public functions of a library. But since `Input::run` (and `Input::run_output`) are designed to panic, I think it's probably best to have the panic source location match the **call-site** of these methods.